### PR TITLE
fix(mcdu): Disable alternate time predictions if fuel manually specified

### PR DESCRIPTION
**Pull request by BravoMike99**
Fixes #[issue_no]

## Summary of Changes

Alternate time predictions become dashed in FUEL PRED & INIT FUEL PRED pages if fuel is specified manually.
## Screenshots (if necessary)

Old & new 
FUELPRED
FuelPred2

INITFUELPREDOLD
INIT FUEL PRED

## References

H3/H4 FCOM
https://youtu.be/efQlfF87c8c?t=205

## Additional context

Discord username (if different from GitHub):
bruno_pt99
## Testing instructions

Manually specify a alternate fuel value in the INIT B & and make sure the time prediction becomes dashed then delete the value and verify that the prediction comes back.
Manually specify an alternate fuel value in the  FUEL PRED Page ( only available after engines running)  and verify that the time at the top of the page and next to the fuel prediction becomes dashed.
Clear the manually inserted value and make sure time becomes available on both fields

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js [hunk 1]
```diff
@@ -2155,7 +2155,7 @@ class FMCMainDisplay extends BaseAirliners {
             if (this.isAltFuelInRange(value)) {
                 this._routeAltFuelEntered = true;
                 this._routeAltFuelWeight = value;
-                this._routeAltFuelTime = FMCMainDisplay.minutesTohhmm(A32NX_FuelPred.computeUserAltTime(this._routeAltFuelWeight * 1000, 290));
+                this._routeAltFuelTime = 0;
```
Comment (tracernz) [inline fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js:2158]:
```suggestion
                this._routeAltFuelTime = null;
```

File: fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js [hunk 2]
```diff
@@ -2242,7 +2242,7 @@ class FMCMainDisplay extends BaseAirliners {
             const deviation = (this.zeroFuelWeight + this._routeFinalFuelWeight - A32NX_FuelPred.refWeight) * A32NX_FuelPred.computeNumbers(airDistance, placeholderFl, A32NX_FuelPred.computations.CORRECTIONS, true);
             if ((20 < airDistance && airDistance < 200) && (100 < placeholderFl && placeholderFl < 290)) { //This will always be true until we can setup alternate routes
                 this._routeAltFuelWeight = (A32NX_FuelPred.computeNumbers(airDistance, placeholderFl, A32NX_FuelPred.computations.FUEL, true) + deviation) / 1000;
-                this._routeAltFuelTime = A32NX_FuelPred.computeNumbers(airDistance, placeholderFl, A32NX_FuelPred.computations.TIME, true);
+                this._routeAltFuelTime = this._routeAltFuelEntered ? 0 : A32NX_FuelPred.computeNumbers(airDistance, placeholderFl, A32NX_FuelPred.computations.TIME, true);
```
Comment (tracernz) [inline fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js:2245]:
```suggestion
                this._routeAltFuelTime = this._routeAltFuelEntered ? null : A32NX_FuelPred.computeNumbers(airDistance, placeholderFl, A32NX_FuelPred.computations.TIME, true);
```

Comment (BravoMike99) [reply fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js]: Both changed to null :)

**Review by tracernz — APPROVED** [review]

**Comment by auroraisluna** [comment]
QA Report

Discord: alepouna
Object of testing: #8273
Tier of Testing : 1
Date : 02/11/2023

Testing Process:
- [PASS] Manually specify a alternate fuel value in the INIT B & and make sure the time prediction becomes dashed then delete the value and verify that the prediction comes back.
- [PASS] Manually specify an alternate fuel value in the FUEL PRED Page ( only available after engines running) and verify that the time at the top of the page and next to the fuel prediction becomes dashed.
- [PASS] Clear the manually inserted value and make sure time becomes available on both fields

Testing Results:
Passed

**Pull request merged**